ios/Podfile

```diff
@@ -12,7 +12,7 @@ end
 target 'Plugin' do
   capacitor_pods
 
-  pod 'Sentry', '7.23.0'
+  pod 'Sentry', '7.27.1'
```

Comment:
the corresponding changes in Podfile.lock are missing from the PR

Reply:
Indeed, thanks for pointing out! I have fixed on the same PR the `require_relative` on the podfiles. With the wrong path the podfiles.lock wasn't updated.